gallery/visualtests/nxTextInput.js

```diff
@@ -28,7 +28,7 @@ describe('NxTextInput', function() {
   });
 
   describe('Simple NxTextInput', function() {
-    it('has a light border when pristine', simpleTest(simpleComponentSelector));
+    it('has a dark border when pristine', simpleTest(simpleComponentSelector));
```

Comment:
Similar to what I said on the `NxFormSelect` tests, there no longer needs to be a distinction between this test and the (mis-named) "non-empty" test.